authority-portal-backend/authority-portal-quarkus/src/test/kotlin/de/sovity/authorityportal/web/tests/services/connector/ConnectorManagementApiServiceTest.kt

```diff
@@ -23,11 +23,7 @@ import de.sovity.authorityportal.db.jooq.enums.ConnectorContractOffersExceeded
 import de.sovity.authorityportal.db.jooq.enums.ConnectorDataOffersExceeded
 import de.sovity.authorityportal.db.jooq.enums.ConnectorOnlineStatus
 import de.sovity.authorityportal.db.jooq.enums.ConnectorType
-import de.sovity.authorityportal.seeds.utils.ScenarioData
-import de.sovity.authorityportal.seeds.utils.ScenarioInstaller
-import de.sovity.authorityportal.seeds.utils.dummyDevConnectorId
-import de.sovity.authorityportal.seeds.utils.dummyDevOrganizationId
-import de.sovity.authorityportal.seeds.utils.dummyDevUserUuid
+import de.sovity.authorityportal.seeds.utils.*
```

Comment:
No asterisk imports pls (it's time i steal the .editorconfig from CP to enforce this)
<img width="984" alt="image" src="https://github.com/user-attachments/assets/5899db5f-557c-4fc9-9dad-5729dd3e2ac7">